Formula/b/bzt.rb

```diff
@@ -23,15 +23,12 @@ class Bzt < Formula
   depends_on "pkg-config" => :build
   depends_on "rust" => :build
   depends_on "libcython"
```

Comment:
I opted to leave in `libcython` as a formula rather than vending the `Cython` package directly.

I _was_ able to build and test it successfully when vending (and could avoid the `.pth` addition for the external formula). Any thoughts for reviewers as to whether we should leave `cython` as a standalone formula as I have done here in this PR?

Reply:
Let's keep it standalone for the time being (we can discuss this on slack, we might need to do an extra pass for that one later).